release-calendar/src/client/components/Hook.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ReleaseCard} from './ReleaseCard';
+import {usePopper} from 'react-popper';
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Since React is loaded from a script tag, we won't need to import this, according to https://reactjs.org/docs/react-dom.html

Reply:
I'm confused. What should i do instead then?

Reply:
I get the warning "'ReactDOM' refers to a UMD global, but the current file is a module. Consider adding an import instead." when I just delete the import line. So yes, it is a global but doesn't that still mean we have to import it?

Reply:
Hmm okay not sure what the warning means. Not a big deal though, resolving.